[docs]: Update Prometheus docs with security info (#2246)
Unleash/unleash: website/docs/api/internal/internal-backstage-api.md

`GET http://unleash.host.com/internal-backstage/prometheus`

> Unleash uses Prometheus internally to collect metrics. These are available on the given url if the `serverMetrics` option is enabled (default=true).

Reviewer: I know you haven't really touched this part, but it read a bit awkwardly to me, so I'll add a suggestion for you here. What do you think?

```suggestion
Unleash uses Prometheus internally to collect metrics. By default, the metrics are available at `/internal-backstage/prometheus`. You can disable this endpoint by setting the `serverMetrics` option to `false`.
```

> Note that it's not recomended to expose Prometheus metrics to the public. You should therefore drop all external access to `/internal-backstage/*` on the network layer to keep your instance secure. 

Reviewer: I think adding this bit is a very good suggestion! 💯  However, I'd like to ask you to add a little bit more nuance to it to make it easier to read.

A couple things:
1. do you have a reference for why it's not recommended to expose prometheus metrics? Or can you explain why it's not recommended? It might be obvious to you, but it's not to everyone else.
2. "You should therefore drop all external access to `/internal-backstage/*` on the network layer to keep your instance secure." Could this be something like: 
3. 
```suggestion
Note that it's not recommended to expose Prometheus metrics to the public. Thus, if you want to keep metrics enabled, you should block all external access to `/internal-backstage/*` on the network layer to keep your instance secure. 
```

What do you think?
